velox/docs/functions/presto/array.rst

```diff
@@ -284,6 +284,16 @@ Array Functions
 
     Flattens an ``array(array(T))`` to an ``array(T)`` by concatenating the contained arrays.
 
+.. function:: ngrams(array(T)) -> array(array(T))
```

Comment:
n parameter is missing

Would be nice to describe the constrains on 'n'. Can it be negative or zero? What happens if array is shorter than n?